src/components/FullDetailView/HeaderSection.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+import React from 'react'
+import { Row, Button, Col } from 'react-bootstrap'
+import isEmpty from 'lodash/isEmpty'
+import { getBadgeUrl } from '../../api/clearlyDefined'
+import Definition from '../../utils/definition'
+import ButtonWithTooltip from '../Renderers/ButtonWithTooltip'
+
+const HeaderSection = props => {
+  const { definition, modalView, changes, renderContributeButton, handleClose, handleSave, handleRevert } = props
+  const { item } = definition
+  const scores = Definition.computeScores(item)
```

Comment:
is it still called `computeScores`? By now the service computes these and the values come back from the service in the definitions

Reply:
I've to check this, this was an implementation of yours, related to show the Score image